### ISSUE TITLE
Calendar Event single-tap (click) not working on mobile devices.

### ISSUE THREAD

**Issue by joshbrekke** [nhn/tui.calendar#1358]
## Version
The latest...
<link rel="stylesheet" href="https://uicdn.toast.com/calendar/latest/toastui-calendar.min.css" />
<script src="https://uicdn.toast.com/calendar/latest/toastui-calendar.min.js" type="text/javascript"></script>

## Test Environment
All browsers appear to work as expected.
Mobile devices such as tablets and phone don't.

Please view jsFiddle below on tablet or phone.

## Current Behavior
When I click on a calendar event from a computer, the alert appears as expected.  However, when interacting with the calendar on a mobile device such as a tablet or phone, I have to tap the event twice to get the alert to appear.

**jsFiddle Link:**
https://jsfiddle.net/f1bkmcne/

```js
// Write example code
```

## Expected Behavior
Single-tap behavior instead of double-tap behavior on mobile devices.

**Comment by adhrinae** [comment]
@joshbrekke 

TL;DR: `tui.calendar` doesn't support mobile devices.

Looks like it's similar to #1270. Please see [this comment](https://github.com/nhn/tui.calendar/issues/1270#issuecomment-1234939980).

**Comment by stanverbakel** [comment]
I have found a workaround for this issue:

```
tuiCalendar.on('afterRenderEvent', (event) => {
    target.addEventListener('click', () => { console.log(event) });
}    
```

You need to have one of the latest versions for this workaround since a issue with afterRenderEvent is only recently fixed: https://github.com/nhn/tui.calendar/issues/1327
